### PR TITLE
Chore/create run

### PR DESCRIPTION
**Pull request by jimid27**
- Relates: https://github.com/PrefectHQ/platform/issues/5625
- Creates shareable GHA for updatecli runs since major and minor are identical just with different manifests and schedules
- Example run: https://github.com/PrefectHQ/flows/actions/runs/7391218314/job/20107588255

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,44 @@
-# actions-updatecli-minor-run
-Actions that runs updatecli, commits those changes if they exist, and opens a PR
+# actions-updatecli-apply
+## Description
+Action that runs updatecli, commits those changes if they exist, and opens a PR
+
+## Inputs
+| Input | Desription | Required |
```
Comment (parkedwards) [inline README.md:6]:
```suggestion
| Input | Description | Required |
```

File: action.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+          echo "changes=false" >> $GITHUB_OUTPUT
+          exit 0
+        else
+          echo "changes=true" >> $GITHUB_OUTPUT
+        fi
+        git add .
+        git commit -m "dependency-version-${{ inputs.run-type }}-${{ steps.date.outputs.date }}"
+        git push --set-upstream origin "dependency-version-${{ inputs.run-type }}-${{ steps.date.outputs.date }}"
+      env:
+        GITHUB_TOKEN: ${{ github.token }}
```
Comment (parkedwards) [inline action.yaml:50]: does this need to be passed in as an action var? i forget if custom actions have access to the workflow's token scope

Comment (jimid27) [reply action.yaml:50]: lemme verify you may be right there

**Review by iotmani — APPROVED** [review]
LGTM!

**Review by parkedwards — APPROVED** [review]
pending the github token thing

File: action.yaml (hunk 2 above)
Comment (jimid27) [reply action.yaml]: https://docs.github.com/en/actions/security-guides/automatic-token-authentication#example-1-passing-the-github_token-as-an-input passing like this example.  May be unnecessary but worth it to be confident the right perms are passing

**Pull request merged**